roles/FortiManager-Playbook-c064238d-8049-4c5b-b3e8-e3a1c41178fe.json

```diff
@@ -0,0 +1,531 @@
+{
+    "@context": "\/api\/3\/contexts\/Role",
```

Comment:
@fortijames Please include only the custom modules that are added with this solution pack in the role. Exclude Out-of-the-Box (OOB) modules such as agents, alerts, and so on.